docs/ecosystem-user-event.mdx

```diff
@@ -615,22 +615,27 @@ method.
 Usage example:
 
 ```jsx
-import React from 'react'
+import React, { useState } from 'react'
 import {render, screen} from '@testing-library/react'
 import userEvent, {specialChars} from '@testing-library/user-event'
 
+const InputElement = () => {
+  const [currentValue, setCurrentValue] = useState('This is a bad example');
+  return <div>
+    <label htmlFor='my-input'>Example:</label>
+    <input id='my-input' type='text' value={currentValue} onChange={(e) => setCurrentValue(e.target.value)} />
+  </div>;
+}
+
 test('delete characters within the selectedRange', () => {
   render(
-    <div>
-      <label htmlFor="my-input">Example:</label>
-      <input id="my-input" type="text" value="This is a bad example" />
-    </div>,
+    <InputElement/>,
   )
   const input = screen.getByLabelText(/example/i)
   input.setSelectionRange(10, 13)
   userEvent.type(input, `${specialChars.backspace}good`)
 
-  expect(input).toHaveValue('This is a good example')
+  expect(screen.getByRole('textbox')).toHaveValue('This is a good example')
```

Comment:
I think we can still use the `input` variable we created above.. Using `screen.getByRole('textbox')` takes into account that our reader knows that `input` has an implicit aria role..

Reply:
done :)